.github/workflows/production.yml

```diff
@@ -66,11 +56,9 @@ jobs:
             - name: Install dependencies
               run: yarn
 
-            - name: Build libraries and contracts
-              run: yarn build
-
+              # test turbo task(s) dependsOn build tasks, no need to explicitly build before testing
             - name: Test libraries, contracts and circuits
-              run: yarn test:${{ matrix.type }}
+              run: yarn test
```

Comment:
Sorry, just noticed this. Why did you remove `:${{ matrix.type }}`.

Reply:
isn't matrix used to run 3 jobs: `yarn test:circuits`, `yarn test:contracts`, `yarn test:libraries`?
and is therefore equivalent to the new `test` script https://github.com/privacy-scaling-explorations/zk-kit/blob/25e11c1c01c050982e3ca33acb7ef17152dd7e10/package.json#L12 (which will run `test:libraries` `test:circuits` and `test:contracts` (=`turbo test:coverage`) in parallel.

Could not test it as the trigger is a push on main...
So if unsure I can revert this.

Reply:
actually let me test this on a new branch

Reply:
you're right, this is wrong. This runs `yarn test` 3 times (so 3x test:contracts, 3x test:circuits, 3x test:libraries). It does not fail but it is just  triggering more jobs for nothing.
https://github.com/privacy-scaling-explorations/zk-kit/actions/runs/8572445780?pr=236

Reply:
reverted in 20ef3838b3dc60c86345239e284389d57bc85c26